src/pybind/mgr/dashboard/frontend/src/app/core/navigation/navigation/navigation.component.ts

```diff
@@ -10,6 +10,12 @@ import {
 } from '../../../shared/services/feature-toggles.service';
 import { SummaryService } from '../../../shared/services/summary.service';
 
+enum SubMenu {
+  cluster,
+  block,
+  rgw
+}
```

Comment:
I think all the navbar/sidebar structure (tree-like) should remain defined in the HTML template, and the component logic should be unaware of that concrete structure (whether the specific "block" or "rgw" or "cluster" menus have submenus, but whether any menu has submenus).